plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreDialect.java

```diff
@@ -676,6 +680,10 @@ public void addExtraKeywords(String... keywords) {
     public void addExtraFunctions(String... functions) {
         super.addFunctions(Arrays.asList(functions));
     }
+    
+    public void addDataTypes(String... dataTypes) {
+        super.addDataTypes(Arrays.asList(dataTypes));
```

Comment:
But what is the reason to override it here without modifications?

Reply:
I think it's more readable this way, and the function for adding functions and extra keywords also have the method overloaded like this, do you still want me to remove it?

Reply:
Yes, please. It is unnecessary.